models/marts/ml/train_predict/predict_position.py

```diff
@@ -20,7 +20,7 @@
 # and that will be used during prediction
 FEATURE_COLS = [
            "RACE_YEAR"
-           ,"CIRCUIT_NAME"
+           ,"RACE_NAME"
```

Comment:
So a really cool MLOps issue came up here: the previous model already sitting in my stage had old naming `CIRCUIT_NAME` and threw errors on a fresh `dbt build `. I had to drop the existing model in my snowflake stage and rebuild it with the new `RACE_NAME` column. Since we don't want to retrain every time, but pick up naming changes I wonder if there is a clever way to version control. I think this is out of scope for now, but a common problem (also if you want to add columns)